src/commcare_cloud/environment/secrets/secrets.yml

```diff
@@ -77,6 +77,15 @@
   legacy_namespace: localsettings_private
   default: ''
   default_overrides_falsy_values: yes
+- name: GOOGLE_OAUTH_CLIENT_ID
```

Comment:
@dannyroberts is this all that's needed to add new secrets?

Reply:
Here are the most up to date instructions for adding new secrets: https://confluence.dimagi.com/display/saas/How+to+add+new+secrets+or+other+local+settings. I see there's also the step of updating tests accordingly for example. But yeah following that should be what you need.